Src/FluentAssertions/Collections/GenericCollectionAssertions.cs

```diff
@@ -1299,7 +1299,7 @@ public AndConstraint<TAssertions> HaveCountGreaterThanOrEqualTo(int expected, st
                 .Then
                 .Given(subject => subject.Count())
                 .ForCondition(actualCount => actualCount >= expected)
-                .FailWith("but found {0}.", actualCount => actualCount)
+                .FailWith("but found {0}: {1}.", Subject.Count(), Subject)
```

Comment:
🤔 This will cause another count of the subject. The original `FailWith` method provided that count through a Func providing the `actualCount` that was calculated on the `Given` line. Same for all the other overloads.

Reply:
Thanks for the feedback.
Updated release notes and modified the `FailWith` usage.